### ISSUE TITLE
Discuss testing Netty on Open Liberty

### ISSUE THREAD

**Issue by isaacrivriv** [OpenLiberty/open-liberty#20876]
Discuss options of testing Netty changes in OL. Asking around it seems like nothing currently exists for enabling beta functionality which is really what we need to test the changes since everything will be beta fenced. 

Potential option is to add a repeat action that creates a jvm.options file in the servers enabling the beta mode which could then be reused in the FATs.

**Comment by isaacrivriv** [comment]
Testing the Netty changes would imply that we would need to run entire FAT buckets using beta mode for the changes to take effect. Discussed a bit about this [here](https://ibm-cloud.slack.com/archives/C31DW78RG/p1650299378548879). Currently there is no way we can do this without manually setting the beta property in some ways. Currently I found we can:

- Create jvm.options for specific server
- Use setJvmOptions with liberty server API class
- Use System.setProperty to set beta mode

The current options we have are all on a class by class or server by server basis meaning that we would have to manually update all the classes or servers to use beta mode and find a way also to revert the changes to test without beta mode also. 

Discussing with the team, we decided to go in a repeat action approach that if used will go through the servers and clients setting the jvm.options to use beta enabled and undo the changes after running. This way we would only need to add a repeat action to the strategic FATs that we want to test Netty with and would only need to write logic for this once and shared between the components as needed. Currently working on a draft with the changes [here](https://github.com/OpenLiberty/open-liberty/pull/20906/files)

For a workaround in the meantime while this is delivered, if beta fencing is already on one can run an entire bucket as is locally by setting the `_JAVA_OPTIONS` env variable with beta turned on e.g. on Mac
```
export _JAVA_OPTIONS=-Dcom.ibm.ws.beta.edition=true
```
